admin/class-admin.php

```diff
@@ -39,8 +36,11 @@ function __construct() {
 			add_action( 'delete_category', array( $this, 'schedule_rewrite_flush' ) );
 		}
 
-		$this->page_gsc = new WPSEO_GSC();
-		$this->dashboard_widget = new Yoast_Dashboard_Widget();
+		$this->admin_features = array(
+			'gsc'              => new WPSEO_GSC(),
```

Comment:
Instead of creating these dependencies in the constructor, could these be injected in to the constructor when an object from this class is created?